shotgun_api3/lib/xmlrpclib.py

```diff
@@ -289,40 +290,60 @@ def __repr__(self):
 
 if _bool_is_builtin:
     boolean = Boolean = bool
-    # to avoid breaking code which references xmlrpclib.{True,False}
-    True, False = True, False
+    try:
+        # TODO: remove this when py2 support is completely dropped
+        # to avoid breaking code which references xmlrpclib.{True,False}
+        True, False = True, False
+
+    except SyntaxError:
+        pass  # We're in py3
+
 else:
     class Boolean:
         """Boolean-value wrapper.
-        
+
         Use True or False to generate a "boolean" XML-RPC value.
         """
-        
+
         def __init__(self, value = 0):
             self.value = operator.truth(value)
-        
+
         def encode(self, out):
             out.write("<value><boolean>%d</boolean></value>\n" % self.value)
-        
+
         def __cmp__(self, other):
+            # cmp() is not part of py3: https://python-future.org/compatible_idioms.html#cmp
             if isinstance(other, Boolean):
                 other = other.value
-            return cmp(self.value, other)
-        
+
+            if self.value is None:
+                return 0 if other is None else -1
+
+            if other is None:
+                return 1
+
+            return self.value.__cmp__(other)
```

Comment:
`cmp()` is gone, this code snippet makes things work exactly like before in py2.

In py3 however -> I think defining `__cmp__()` is a no-op